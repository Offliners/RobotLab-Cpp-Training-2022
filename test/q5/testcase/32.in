(12/11+20/19)-9=*(13-(8+11))=+6*(15-19)/3-(14+7)+9/3-7/5+4-(14-6)/12+18-7*18=/((8-16)+(19*10)/20)=
