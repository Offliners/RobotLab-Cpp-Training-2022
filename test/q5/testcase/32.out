-7
42
-89
-89
